source/draw/gpu/GpuContext.ooc

```diff
@@ -50,7 +50,7 @@ GpuContext: abstract class {
 	getCurrentIndex: func -> Int { 0 }
 	alignWidth: virtual func (width: Int, align := AlignWidth Nearest) -> Int { width }
 	isAligned: virtual func (width: Int) -> Bool { true }
-	packToRgba: abstract func (source: GpuImage, target: GpuBgra, viewport: IntBox2D)
+	packToRgba: abstract func (source: GpuImage, target: GpuBgra, viewport: IntBox2D, padding: Float)
```

Comment:
Create two separate functions instead, one with padding as parameter and one without. The padding should be an Integer and represent the number of bytes padded.